packages/block-library/src/post-comments-form/index.php

```diff
@@ -45,11 +45,17 @@ function render_block_core_post_comments_form( $attributes, $content, $block ) {
 	// to the block is carried along when the comment form is moved to the location
 	// of the 'Reply' link that the user clicked by Core's `comment-reply.js` script.
 	$form = str_replace( 'class="comment-respond"', $wrapper_attributes, $form );
+	$form = str_replace( '</form>', '<div style="color: red;" data-wp-text="state.core.commentsFormError"></div></form>', $form );
```

Comment:
FWIW, there's a [`comment_form_after` action](https://developer.wordpress.org/reference/hooks/comment_form_after/) that can be used to render stuff here. (Though I'm not sure it's worth using it here as it'd require adding the action before the `comment_form()` call and removing it after.)

Reply:
On second thought -- if we're not married to this exact location (i.e. right before the `</form>`, which means after the 'Submit' button), we might also use one of the (many) options to the [`comment_form` function](https://developer.wordpress.org/reference/functions/comment_form/). There is e.g. `comment_notes_after` which is rendered after the `textarea` (before the 'Submit' button), and which defaults to the empty string.

Reply:
The snackbar seems like a great idea. The message would also have to get announced when using assistive technologies. In both cases when it’s successful, or it fails.

Reply:
I like the snackbar idea! We can try it to see how it feels and to see how easy it would be to create one with the Interactivity API 🙂